### PR TITLE
Chore/create staging env

### PR DESCRIPTION
**Pull request by giovanneguerra**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204058949837552

### CODE REVIEW

File: apps/explorer/tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
     "target": "ES2022",
     "module": "ES2022",
     "useDefineForClassFields": false,
-    "lib": ["ES2022", "dom"]
+    "lib": ["ES2022", "dom"],
+    "paths": {
+      "tslib": ["./node_modules/tslib/tslib.d.ts"]
+    }
```
Comment (smakhtin) [inline apps/explorer/tsconfig.json:25]: Are we sure that tslib definition is needed?

**Review by smakhtin — CHANGES_REQUESTED** [review]
@giovanneguerra PR looks good, didn't understand why layout was changed (no attributes panel), please see comments on Vercel.

File: apps/explorer/tsconfig.json (hunk 1 above)
Comment (giovanneguerra) [reply apps/explorer/tsconfig.json:25]: Yes, or else we gonna have problems with the linter complaining about it.

**Comment by giovanneguerra** [comment]
> 
It was my mistake, I forgot to uncomment properties, will push the fix

**Review by smakhtin — APPROVED** [review]

**Pull request merged**